patches/minecraft/net/minecraft/entity/ai/attributes/GlobalEntityTypeAttributes.java.patch

```diff
@@ -1,15 +1,24 @@
 +   private static final Map<EntityType<? extends LivingEntity>, AttributeModifierMap> FORGE_ATTRIBUTES = new java.util.HashMap<EntityType<? extends LivingEntity>, AttributeModifierMap>();
-+   
+ 
++   @Deprecated // FORGE: Use net.minecraftforge.event.entity.EntityAttributeCreationEvent#put
 +   public static AttributeModifierMap put(EntityType<? extends LivingEntity> type, AttributeModifierMap map) {
 +       return FORGE_ATTRIBUTES.put(type, map);
 +   }
- 
++
++   @Deprecated // FORGE: Use net.minecraftforge.event.entity.EntityAttributeModificationEvent#add
++   public static AttributeModifierMap combineWithExisting(EntityType<? extends LivingEntity> livingEntity, AttributeModifierMap.MutableAttribute toCombine) {
++      AttributeModifierMap modifier = func_233835_a_(livingEntity);
++      AttributeModifierMap.MutableAttribute newMutable = modifier != null ? new AttributeModifierMap.MutableAttribute(modifier) : new AttributeModifierMap.MutableAttribute();
++      newMutable.combine(toCombine);
++      return FORGE_ATTRIBUTES.put(livingEntity, newMutable.func_233813_a_());
++   }
```

Comment:
Move this to ForgeHooks with a major *INTERNAL ONLY* warning.
Move FORGE_ATTRIBUTES to Forge as well, it's private it can change without us caring about breaking things.
Bounce put to a ForgeHooks function that logs a error in the console about deprecated call {Make it a small one liner because people will complain, but it'll just dissapear in 1.17}